dropbox-sdk-dotnet/Dropbox.Api/DropboxClient.cs

```diff
@@ -32,11 +32,21 @@ public DropboxClient(string oauth2Token)
         /// <summary>
         /// Initializes a new instance of the <see cref="T:Dropbox.Api.DropboxClient"/> class.
         /// </summary>
-        /// <param name="oauth2AccessToken">The oauth2 access token for making client requests.</param>
```

Comment:
Should we keep this old constructor?

Reply:
I think it is fine to deprecate this constructor.  We already have labeled it as deprecated and I think the refresh, app key combo will be a widely used constructor (slt & PKCE)